packages/create-bison-app/cli.js

```diff
@@ -54,35 +55,40 @@ function generateQuestions(appName) {
     {
       name: "db.dev.name",
       type: "input",
-      message: "What is the local database name?",
+      prefix: "The following link <link> will help you setup a new Postgres database if needed.\n\bThe name you will use for the new database (or current).\n",
```

Comment:
What do you think about asking if they have Postgres setup first?

> Do you have Postgres setup? y/n

(If the user answers no)
> Visit https://github.com/echobind/bisonapp/blob/main/docs/postgres.md for instructions!

(If the user answers yes, prompt for the database info below).

I like that you added both Homebrew and Postgres.app docs. Could we create a parent `Postgres` doc that links to each of those? That way, we can point users to a single place and they can choose how to proceed.